internal/dag/builder_test.go

```diff
@@ -1424,7 +1424,7 @@ func TestDAGInsertGatewayAPI(t *testing.T) {
 				},
 			),
 		},
-		"HTTPRoute references a backend in a different namespace, no ReferencePolicy": {
+		"HTTPRoute references a backend in a different namespace, no ReferenceGrant": {
```

Comment:
I decided to just switch these tests over rather than duplicate them, since the old ones will go away soon, but I can do the former if folks think it's necessary.